docs/source/API/core/view/create_mirror.md

```diff
@@ -44,8 +48,16 @@ template <class Space, class ViewType>
 ImplMirrorType create_mirror_view(decltype(Kokkos::ViewAllocateWithoutInitializing()),
                                   Space const& space, ViewType const&);
 
+template <class ViewType, class... ViewCtorArgs>
+auto create_mirror_view(Impl::ViewCtorProp<ViewCtorArgs...> const& arg_prop,
```

Comment:
Can we try something like:

```c++
template <class ViewType, class... ViewCtorArgs>
 auto create_mirror_view(ALLOC_PROP const& arg_prop, ViewType const& v);
```

or 

```c++
template <class ViewType, class... ViewCtorArgs>
 auto create_mirror_view(/*see below*/ const& arg_prop, ViewType const& v);
```

or

```c++
template <class ViewType, class... ViewCtorArgs>
 auto create_mirror_view(/* is-alloc-prop */ const& arg_prop, ViewType const& v);
```

Reply:
I think each of those is better than either `Impl::ViewCtorProp`, or `decltype(ViewAllocationWithoutInitializing())`

You then in the description define what `ALLOC_PROP` or `/*is-alloc-prop*/` means.

Reply:
I think right now it seems we have a preference for the last one? Maybe as `/*is-view-alloc-prop*/` to make it even more clear? And then spell out what all the things are which can go there?

Reply:
Note I do not want to use this in places of the explicit "takes an execution space instance as first argument" overloads.

Reply:
+1 for option 3

Reply:
Here you go!